class.jetpack.php

```diff
@@ -757,7 +716,68 @@ private function __construct() {
 		if ( ! has_action( 'shutdown', array( $this, 'push_stats' ) ) ) {
 			add_action( 'shutdown', array( $this, 'push_stats' ) );
 		}
+	}
+
+	function setup_xmlrpc_handlers( $request_params, $is_active, $is_signed, Jetpack_XMLRPC_Server $xmlrpc_server = null ) {
```

Comment:
should we move this to the connection package?

Reply:
What do you say we move everything into the connection package in #12699 later? This block depends on a lot of static methods that I don't feel comfortable moving right before the code freeze.